syntax = "proto3";

// Corresponds to struct Clip.
message ClipProto {
	int64 pts_in = 1;
	int64 pts_out = 2;
	repeated string description = 3;
	int64 stream_idx = 4;
}

message ClipListProto {
	repeated ClipProto clip = 1;
}

message StateProto {
	ClipListProto clip_list = 1;
	ClipListProto play_list = 2;
}
